s3path.py

```diff
@@ -818,7 +818,24 @@ def key(self):
         return key
 
     @classmethod
-    def from_bucket_key(cls, bucket, key, version_id=None):
+    def from_bucket_key(cls, bucket):
+        """
+        from_bucket_key class method create a class instance from bucket, key pair's
+
+        >> from s3path import PureS3Path
+        >> PureS3Path.from_bucket_key(bucket='<bucket>', key='<key>')
+        << PureS3Path('/<bucket>/<key>')
+        """
+        bucket = cls(cls._flavour.sep, bucket)
+        if len(bucket.parts) != 2:
+            raise ValueError(f'bucket argument contains more then one path element: {bucket}')
+        key = cls(key)
+        if key.is_absolute():
+            key = key.relative_to('/')
+        self = bucket / key
+        return self
```

Comment:
even tho this method doesn't allow you to give a version ID we still want to set the version ID attribute of the `S3Path` object to `None`. We could do that the following way:
```python
self = bucket / key
self.version_id = None
return self
```